siemens_data.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+import pandas as pd
+
+
+class SiemensData:
+    """ Class for reading and manipulating Siemens data from transformed CSVs.
```

Comment:
So are we thinking this is one of the readers or some of the output we might want from our API?